### ISSUE TITLE
Database error on latest rebase

### ISSUE THREAD

**Issue by tim-pearce** [tbar0970/jethro-pmm#786]
I rebased my dev copy of Jethro to the latest version.
I got 
SYSTEM ERROR
PDO::quote(): Passing null to parameter #2 ($type) of type int is deprecated
Line 94 of File /Data/htdocs/jethro/jethro-pmm/include/jethrodb.php

I have updated my local version to :-
```
public function quote($string, $paramtype = NULL)
	{
		if ($string === NULL) {
			return 'NULL';
		} else {
			if ($paramtype === NULL) {
				return parent::quote($string);
			} else {
				return parent::quote($string, $paramtype);
			}
		}
	}
```

**Comment by tbar0970** [comment]
Hi Tim, just checking:  is this happening against the latest *release* or
the head of git? I think I upgraded the dependencies since doing the latest
jethro release, so that might be the cause.

**Comment by tim-pearce** [comment]
It's the head version but I think the issue is PHP 8(.1).

I've got an error regarding $GLOBALS from the TBS code (used by document merge) and in my document merge code where it's complaining that the third parameter can't be null - very similar to this issue.

I will check the TBS site to see if there are changes for PHP 8.1

I'm running my dev system with Artix Linux, which is a systemd derivative or Arch Linux so I'm close to the 'bleeding edge' of any changes to things like PHP.

**Comment by tim-pearce** [comment]
The latest version of TBS seems to fix the problem with TBS.

**Comment by tbar0970** [comment]
REsolved in #787
